### PR TITLE
Iteration three

### PR DESCRIPTION
**Pull request by zliibbe**
## Purpose:
This PR provides the login screen for the website and completes Iteration 3. It also updates the project's README.md

*What does this merge do? Check all that apply.*
- [x] New feature (non-breaking change which adds functionality)
- [ ] Improves an existing feature
- [ ] Fixes a bug
      - Bug fixed:
- [ ] Refactor
      - Code refactored:
- [x] Documentation update

## Description: What does this PR do?
This PR provides a login screen for the site which can be accessed via `traveler` with a number between 1-50 and password of `traveler`. Once the user logs in, the user's information is displayed.

## Instructions for reviewer:
*Where should the reviewer start?* See HTML for login page and scripts.js and domUpdates. js for login functionality
*How should this be manually tested?*
*Any background context you want to provide?*
*What are the relevant tickets?* closes issue #3 

## Issues:
*Are there any concerns, issues, or bugs in this branch? If so describe them.*
No serious bugs as all functionality is intact. There are break points for smaller screen sizes, which will have to be addressed in a further iteration.

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have used tests to prove my fix is effective or that my feature works
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my change

## Future Steps:

* Iteration 4
* Complete 2 tests in tirip-test.js
* Include breakpoints for responsive design
* See README for further future steps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,105 +1,71 @@
...
+User Dashboard: Once logged in, a user can see all of their trips displayed.  The header also includes the stats on how much they've spent on trips this year.
+
+Book a Trip: A booking form is displayed in the left sidebar in desktop view. Users can fill out this form to request a new trip. Once submitted, the trip shows up in the list of trips with a status of 'pending'. The traveler must wait for the trip to be approved by the Travel Agency (are those still a thing?!).
+
+--- 
+## Contributers
+- Primary contributer: [Zach Liibbe](https://github.com/zliibbe)
+
+- Support with pull requests and code review:
+  - [Andrew Musselman](https://github.com/Andrew-Musselman)
```
Comment (Andrew-Musselman) [inline README.md:48]: Nice shoutout 😊

File: dist/index.html [hunk 2]
```diff
@@ -74,23 +84,22 @@ <h2>Book A New Trip:</h2>
       <option value="50">Hobart, Tasmania</option>
     </select>
     <label for="date">Start Date:</label>
-      <input type="date" name="date" id="date" class="date-input" required>
+      <input type="date" name="date" id="date" class="date-input" min="2022-03-09"required>
```
Comment (Andrew-Musselman) [inline dist/index.html:87]: Oh! I didn’t know you could add a min for the date picker! That’s a good call

Comment (CHaiz15) [inline dist/index.html:87]: Ayyy niice

File: src/scripts.js [hunk 3]
```diff
@@ -23,19 +17,43 @@ const tripNumOfTravelers = document.querySelector('.number-of-travelers-input');
...
-    return Promise.all([fetchDataFor(`travelers`), 
+const hideLogin = () => {
+    domUpdates.toggleHidden(loginSection)
+    domUpdates.toggleHidden(bookSection)
+    domUpdates.toggleHidden(tripSection)
+}
+
+const getLoginInfo = (event) => {
+    event.preventDefault()
+    const traveler = userName.value.slice(0,8);
```
Comment (Andrew-Musselman) [inline src/scripts.js:36]: I like how concise this is

**Review by Andrew-Musselman — APPROVED** [review]
Looks really good!

**Review by CHaiz15 — APPROVED** [review]
👍

**Pull request merged**